src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -629,9 +630,11 @@
 (rf/reg-event-fx :wallet/end-transaction-flow
  (fn [{:keys [db]}]
    (let [address (get-in db [:wallet :current-viewing-account-address])]
-     {:fx [[:dispatch [:dismiss-modal :screen/wallet.transaction-confirmation]]
-           [:dispatch [:wallet/navigate-to-account-within-stack address]]
-           [:dispatch [:wallet/select-account-tab :activity]]]})))
+     {:fx [[:dispatch [:wallet/navigate-to-account-within-stack address]]
+           [:dispatch [:wallet/select-account-tab :activity]]
+           [:dispatch-later
+            [{:ms       20
```

Comment:
Why do we need to delay the cleanup ?

Reply:
As far as I know, this was an existing approach to the delay the cleanup to give time for the screen navigation.